javascript/modules/client/src/internal/interfaces/core.ts

```diff
@@ -31,11 +32,14 @@ export interface IClientIpfsCore {
   fetchBytes: (cid: string) => Promise<Uint8Array | undefined>;
 }
 export interface IClientGraphQLCore {
-  // Add here
+  getClient: () => GraphQLClient
+  shiftClient: () => void
+  isUp: () => Promise<boolean>
+  ensureOnline: () => Promise<void>;
 }
 
 export interface IClientCore {
   web3: IClientWeb3Core;
   ipfs: IClientIpfsCore;
-  graphql: IClientGraphQLCore;
+  subgraph: IClientGraphQLCore;
```

Comment:
General question: Should this be about subgraph or about GraphQL?
Subgraph is the specific service we use not, but GraphQL is the general technology being used.
I guess all appearences should speak about `graphql` and similar

Reply:
This change was to unify the codebase because in some places it was used as `graphql` and in others as `subgraph`. About the naming is weird because in some parts of the code `subgraph` made more sense and in others `graphql`, so in the end i just picked one, but I guess `graphql` makes more sense in general so I'll change it

Reply:
:+1: